src/Compilers/Core/Portable/Syntax/SyntaxNode.cs

```diff
@@ -1392,31 +1392,16 @@ public bool IsEquivalentTo(SyntaxNode node, bool topLevel = false)
         /// Serializes the node to the given <paramref name="stream"/>.
         /// Leaves the <paramref name="stream"/> open for further writes.
         /// </summary>
-        [Obsolete(SerializationDeprecationException.Text, error: false)]
+        [Obsolete(SerializationDeprecationException.Text, error: true)]
```

Comment:
Let's update the note in [https://github.com/dotnet/roslyn/blob/main/docs/Breaking%20API%20Changes.md](https://github.com/dotnet/roslyn/blob/main/docs/Breaking%20API%20Changes.md)

Reply:
done.